.github/workflows/lint.yml

```
@@ -0,0 +1,33 @@
+name: Fix Code Style
+
+on: [push]
```

This looks great! I think that running code style fixes on every push isn't ideal. Maybe let's change this so it only runs on PRs?

```suggestion
on:
  pull_request:
    types: [opened, synchronize]
    paths:
      - '**.php'
      - '.github/workflows/lint.yml'
```